site/content/ytt/docs/latest/lang-ref-ytt-schema.md

```diff
@@ -386,3 +386,31 @@ app_domains:
   - "carvel.dev"
   - 8080
 ```
+
+_Example: Error case when setting schema within an any type fragment_
+
+```yaml
+#@data/values-schema
+---
+#@schema/type any=True
+app_domains:
+  #@schema/default "localhost"
+  #@schema/type any=False
```

Comment:
`#@schema/type any=False` seems weird to me. cc @pivotaljohn

Reply:
weird? how so?